src/nomad_material_processing/vapor_deposition.py

```diff
@@ -194,21 +189,32 @@ class SubstrateTemperature(ArchiveSection):
     )
 
 
-class Substrate(ArchiveSection):
+class SubstrateSetup(PlotSection, ArchiveSection):
     m_def = Section(
         a_plotly_graph_object={
-            "data": {"x": "temperature/duration", "y": "temperature/temperature"},
+            "data": {"x": "temperature/process_time", "y": "temperature/temperature"},
             "layout": {"title": {"text": "Plotly Graph Object"}},
             "label": "Plotly Graph Object",
             "index": 1,
         },
     )
+
+    # which of the folllwoing two?
+
     thin_film = Quantity(
         description="""
         The thin film that is being created during this step.
         """,
         type=ThinFilm,
     )
+    layers = SubSection(
+        description="""
+        An ordered list (starting at the substrate) of the thin films making up the
+        thin film stacks.
+        """,
+        section_def=ThinFilmReference,
+        repeats=True,
+    )
```

Comment:
As we discussed yesterday it should be the thin film that is created in this step.
```suggestion
    thin_film = Quantity(
        description="""
        The thin film that is being created during this step.
        """,
        type=ThinFilm,
    )
```